cypress/utils/utils.ts

```diff
@@ -1758,3 +1759,23 @@ export function manageCredentialsForMultipleApplications(
         currentApp.selectApplication();
     });
 }
+
+export function validateSortBy(sortBy: string) {
+    const unsortedList = getTableColumnData(sortBy);
+
+    // Sort the application inventory by name in ascending order
+    clickOnSortButton(sortBy, SortType.ascending);
+    cy.wait(2 * SEC);
+
+    // Verify that the application inventory table rows are displayed in ascending order
+    const afterAscSortList = getTableColumnData(sortBy);
+    verifySortAsc(afterAscSortList, unsortedList);
+
+    // Sort the application inventory by name in descending order
+    clickOnSortButton(sortBy, SortType.descending);
+    cy.wait(2 * SEC);
+
+    // Verify that the application inventory table rows are displayed in descending order
+    const afterDescSortList = getTableColumnData(sortBy);
+    verifySortDesc(afterDescSortList, unsortedList);
```

Comment:
All the comments are referencing the "Application Inventory" but this function looks like it can handle other tables too, right?

Maybe "Application inventory" should be replaced with "table"

Reply:
updated